content/800-data-platform/050-data-proxy/100-use-data-proxy.mdx

```diff
@@ -212,8 +212,22 @@ The generated Prisma Client has a reduced bundle size because the entire query e
 
 You can adjust the maximum number of database connections of each Data Proxy connection pool.
 
-To do so, in the Prisma Data Platform add the `connection_limit` parameter to the database connection string of the environment in your project.
+To do so, in the Prisma Data Platform, add the `connection_limit` parameter to the database connection string of the environment in your project.
+
+The _default_ value for the `connection_limit` in each Query Engine (QE) instance is _5_. This default value is determined using the calculation (`num_physical_cpus * 2 + 1`), where each QE instance consist of two CPUs.
```

Comment:
> The _default_ value for the `connection_limit` in each Query Engine (QE) instance is _5_.

This is not correct: It depends on the machine on which the QE is running, so this doesn't work as a blanket statement.

```suggestion
The _default_ value for the `connection_limit` in each Data Proxy instance is _5_. This default value is determined using the calculation `num_physical_cpus * 2 + 1`. The machines Data Proxy is running on have two two CPUs, so: `2 * 2 + 1 = 5`.
```

Reply:
Fixing this one!

Reply:
Replaced the statement [here](https://github.com/prisma/docs/pull/4776/commits/fd38ac03dc3aabbfa643c19c4e65e98dfcdb2431)